src/infrastructure/implementations/basic-key-manager.ts

```diff
@@ -0,0 +1,22 @@
+import { TokenSigner } from "jsontokens";
+import { IKeyManager } from "../../core/interfaces/key-manager";
+import { getLogger } from "../../packages/logger";
+import { getKeyPairFromPrivKey } from "../../packages/utils";
+const log = getLogger(__filename);
+export class BasicKeyManager implements IKeyManager {
+    private privateKey: string;
+    private publicKey: string;
+    constructor(privateKey: string) {
+        const keyPair = getKeyPairFromPrivKey(privateKey);
```

Comment:
private key encryption and private key cleanup when not in use are pending.

Reply:
privateKey encryption is done, privateKey cleanup is still pending.

Reply:
PrivateKey cleanup - https://github.com/cruxprotocol/js-sdk/pull/152/commits/dc2b1ed1fcce16e7ad46571bb3a3c77ccfd9ac7c